app/backend/aquifers/migrations/0012_aquifer_geom.py

```diff
@@ -2,6 +2,22 @@
 
 import django.contrib.gis.db.models.fields
 from django.db import migrations
+from aquifers.models import Aquifer as NewAquifer
+
+
+def add_shapefile(apps, schema_editor):
+    Aquifer = apps.get_model('aquifers', 'Aquifer')
+    aquifers = Aquifer.objects.all()
+    aquifer = aquifers[0]
```

Comment:
It looks like this may be for loading test fixtures.  Should it run on production?

Reply:
@stephenhillier how do we prevent a migration running on production?

Reply:
I'm just going to remove it. It's actually commented out anyway :+1: